src/main/java/lv/acodemy/homework/SecondHomeWork.java

```diff
@@ -0,0 +1,52 @@
+package lv.acodemy.homework;
+
+import java.util.Arrays;
+
+public class SecondHomeWork {
+    public static void main(String[] args) {
+
+        String[] countrys = {"Moskow", "Tallin", "Amsterdam", "Vilnus", "Latvija", "Berlin", "Stocholm", "Lisabon", "Praha", "Barcelona"};
+        System.out.println(Arrays.toString(countrys));
+        System.out.println("I love " + countrys[2]);
+
+        for (String euroCity : countrys) {
+            if (euroCity.equals("Amsterdam")) {
+                System.out.println("OooOOOoooo nice city!!!There is a GOOD wead!!!");
+            } else if (euroCity.equals("Latvija")) {
+                System.out.println(countrys[4] + " is my motherland");
```

Comment:
Imagine you don't know where Latvija actually is, do not use countrys[4] here :)